sci-libs/gdal/gdal-2.2.4.recipe

```diff
@@ -40,7 +40,7 @@ if [ $targetArchitecture != x86_gcc2 ]; then
 fi
 
 portVersionCompat="$portVersion compat >= 1.11"
-libVersion=20.3.2
+libVersion=20.3.3
 libVersionCompat="$libVersion compat >= 1.18"
```

Comment:
Shouldn't this be

    libVersionCompat="$libVersion compat >= ${libVersion%%.*}

Or am I missing something?